nf_wochenende.nf

```diff
@@ -217,42 +217,52 @@ workflow {
     // run main Wochenende process
     wochenende(input_fastq_R1)
     
-    // run reporting
-    reporting(wochenende.out.calmd_bam_txts.flatten())
+    if (params.stage_reporting) {
+        // run reporting
+        reporting(wochenende.out.calmd_bam_txts.flatten())
+    }
 
-    // run haybaler
-    haybaler(reporting.out.us_csvs.collect())
+    if (params.stage_haybaler) {
+        // run haybaler
+        haybaler(reporting.out.us_csvs.collect())
+    }
```

Comment:
since haybaler needs the reporting output, we should make sure, reporting was run

Could be something like:

if (params.stage_reporting) or (params.stage_haybaler) {
reporting(input)
}

Reply:
Okay, but right now it could easily fail, e.g. if a user wants haybaler done but doesn't set `stage_reporting = true`...

So we should at leat inform the users in the `nextflow.config`, what other stages one stage requires.

Reply:
Yes, its a valid and good point. However, it can easily make things very difficult to understand, and then buggy. The user sets parameter, but dev logic steps overwrites those parameters, and the users get mad (Tobias and I had this repeatedly with logic in the python script). So I am a little hesitant ... 

Thanks for the review.